build/Dockerfile-ubuntu-18.04-go1.20.6-solc0.8.13

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04 as go_builder
+RUN apt update && apt install -y software-properties-common
+RUN add-apt-repository ppa:longsleep/golang-backports -y && \
+    apt install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt install -y build-essential bash gcc musl-dev openssl wget golang golang-1.17
+RUN wget -O go.src.tar.gz https://dl.google.com/go/go1.20.6.src.tar.gz
+RUN tar -C /usr/local -xzf go.src.tar.gz
+RUN cd /usr/local/go/src/ && \
+    ./make.bash
+
+FROM ubuntu:18.04
+RUN apt update
+RUN apt install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt install -y ca-certificates libboost-all-dev git make gcc libc-dev curl bash python3 python3-dev python3-pip wget unzip software-properties-common
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test &&\
+    apt-get update &&\
+    apt install g++-10 gcc-10 -y &&\
+    update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 90 --slave /usr/bin/g++ g++ /usr/bin/g++-10 --slave /usr/bin/gcov gcov /usr/bin/gcov-10
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install && rm awscliv2.zip
+COPY --from=go_builder /usr/local/go /usr/local
+RUN mkdir /usr/bin/solc && wget -q https://github.com/ethereum/solidity/releases/download/v0.8.13/solidity_0.8.13.tar.gz -O -| tar -xvzf - -C /usr/bin/solc
```

Comment:
This file is not matched with docker hub. https://hub.docker.com/layers/klaytn/build_base/1.2-ubuntu18.04-go.1.20.6-solc0.8.13/images/sha256-cbb3c5bdfe7eb089bb6933c3f3bed2edf90dc05ccddb50ab4bde13c61646c5a5?context=explore
Is it okay?

Reply:
yes @yoomee1313 , I have release it after you said that want to use binary from solc. It's good to go